src/i18n/en.json

```diff
@@ -65,12 +65,19 @@
 			"RESET_SEARCH": "Reset search"
 		}
 	},
-	"MODAL" : {
+	"MODAL": {
 		"CLOSE": "Close modal"
 	},
 	"OVERFLOW_MENU" : {
```

Comment:
space here

Reply:
Good catch! Looks like there's another one of those in there from way back when.